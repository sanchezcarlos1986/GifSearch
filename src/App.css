.App {
  text-align: center;
}

.App-content {
  background-color: #282c34;
  min-height: 100vh;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  font-size: calc(10px + 2vmin);
  color: white;
}

.ListOfGifs {
  display: grid;
  list-style: none;
  padding: 0;
  grid-template-columns: repeat(3, 1fr);
}

.ListOfGifs img {
  max-width: 100%;
}
