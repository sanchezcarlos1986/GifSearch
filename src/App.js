import React from 'react';
import './App.css';
import ListOfGifs from './components/ListOfGifs';
import {Route} from 'wouter';

/**
 * App component
 * @constructor
 * @return {function} Returns a component
 */
function App() {
  return (
    <div className="App">
      <h1>Giffy</h1>
      <Route path="/gif/:keyword" component={ListOfGifs} />
    </div>
  );
}

export default App;
