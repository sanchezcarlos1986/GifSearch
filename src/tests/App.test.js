import React from 'react';
import '@testing-library/jest-dom';
import App from '../App';
import {shallow} from 'enzyme';

describe('App', () => {
  let component;

  beforeEach(() => {
    component = shallow(<App />);
  });

  it('should renders ok', () => {
    expect(component).not.toBeNull();
    expect(component.hasClass('App')).toBe(true);
    expect(component).toMatchSnapshot();
  });
});
