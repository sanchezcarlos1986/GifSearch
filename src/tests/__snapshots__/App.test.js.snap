// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should renders ok 1`] = `ShallowWrapper {}`;
